.circleci/orbs/orb-fsa/@orb.yml

```diff
@@ -1,14 +1,15 @@
 version: 2.1
-description: "WhiteSource Unified Agent is a tool which extracts descriptive information from your open source libraries located on your file system."
+description: "The orb integrates WhiteSource Unified Agent, a tool which extracts descriptive information from your open source libraries located on your file system. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent"
```

Comment:
Please refine to:
description: "The orb integrates WhiteSource Unified Agent, a tool which extracts descriptive information from your open source libraries located on your file system. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent"

Reply:
```suggestion
description: "The orb integrates WhiteSource Unified Agent, a tool which extracts descriptive information from your open source libraries located on your file system. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/33718339/Unified+Agent"
```

Reply:
```suggestion
description: "The orb integrates WhiteSource Unified Agent, a tool for detecting security vulnerabilities and compliance data related to open source components. For more information: https://whitesource.atlassian.net/wiki/spaces/WD/pages/804814917/Unified+Agent+Overview"
```